spikeinterface/core/job_tools.py

```diff
@@ -52,6 +53,10 @@ def split_job_kwargs(mixed_kwargs):
             job_kwargs[k] = v
         else:
             specific_kwargs[k] = v
+    # if n_jobs is -1, set to os.cpu_count()
+    if "n_jobs" in job_kwargs:
+        if job_kwargs["n_jobs"] == -1:
+            job_kwargs["n_jobs"] = os.cpu_count()
```

Comment:
I think this is not a good idea to use this function as a checker.
If we need a checker lets make a specific function

Reply:
I agree. Something like `fix_job_kwargs`, which merges with globals and does the -1 to cpu count operation?